staging/src/k8s.io/apiextensions-apiserver/test/integration/ratcheting_test.go

```diff
@@ -222,19 +222,13 @@ func (u updateMyCRDV1Beta1Schema) Do(ctx *ratchetingTestContext) error {
 		uuidString := string(uuid.NewUUID())
 		// UUID string is just hex separated by dashes, which is safe to
 		// throw into regex like this
-		pattern := "^" + uuidString + "$"
 		sentinelName := "__ratcheting_sentinel_field__"
 		sch.Properties[sentinelName] = apiextensionsv1.JSONSchemaProps{
-			Type:    "string",
-			Pattern: pattern,
-
-			// Put MaxLength condition inside AllOf since the string_validator
-			// in kube-openapi short circuits upon seeing MaxLength, and we
-			// want both pattern and MaxLength errors
-			AllOf: []apiextensionsv1.JSONSchemaProps{
+			Type: "string",
+			XValidations: []apiextensionsv1.ValidationRule{
 				{
-					MinLength: ptr((int64(1))), // 1 MinLength to prevent empty value from ever being admitted
-					MaxLength: ptr((int64(0))), // 0 MaxLength to prevent non-empty value from ever being admitted
```

Comment:
does switching this from a declarative openapi validation to a CEL rule change the ratcheting logic it is exercising?

Reply:
is there a way to keep this declarative and still output the message (like making the sentinal uuid an enum or something) ... just trying to minimize the test change

Reply:
> does switching this from a declarative openapi validation to a CEL rule change the ratcheting logic it is exercising?

This code is not for triggering the validation being tested and not meant to excercise ratcheting logic, this is meant to just observe that the new schema has taken effect, before we continue with testing ratcheting. 

> is there a way to keep this declarative and still output the message (like making the sentinal uuid an enum or something) ... just trying to minimize the test change

Changed from using CEL error message to using enum, which also includes the cases in its output:

```
"MyCoolCRD.mygroup.example.com \"sentinel-resource\" is invalid: __ratcheting_sentinel_field__: Unsupported value: \"invalid-0\": supported values: \"8c1cc36b-9240-4405-8f25-c1d5878eac0d\""
```